gen/calc.py

```diff
@@ -421,7 +425,10 @@ def validate_os_type(os_type):
         },
         'rexray_config_method': {
             'file': {
-                'must': {'rexray_config_contents': calculate_rexray_config_contents},
+                'must': {'rexray_config_contents': calculate_rexray_config_from_file},
+            },
+            'string': {
+                'must': {'rexray_config_contents': calculate_rexray_config_from_string},
             },
```

Comment:
We should unfortunately probably keep / test it then.... Might be worthwile to ping @pyronicide to see if he has any numbers around usage / if it's safe to drop. Easy non-CF AWS makes some sense to have around.